### ISSUE TITLE
0.33 RC1 - Filters in notebook mode render offscreen

### ISSUE THREAD

**Issue by nickhamlin** [metabase/metabase#10537]
**Describe the bug**
In the 0.33 Release Candidate, trying to apply filters in notebook mode on a small display can result in the list of fields extending off the screen with no way to select all options. This is particularly tricky since the lists render upwards, so searching for a field to bring it on screen doesn't work as a workaround

**Logs**
JS logs are here:
```
app-main.bundle.js?1d74d19605fd6afae319:5 Tooltip::componentDidMount: no DOM node for tooltip undefined
value @ app-main.bundle.js?1d74d19605fd6afae319:5
app-main.bundle.js?1d74d19605fd6afae319:5 DEPRECATED: metabase/redux/metadata fetchDatabases
v @ app-main.bundle.js?1d74d19605fd6afae319:5
app-main.bundle.js?1d74d19605fd6afae319:5 DEPRECATED: metabase/redux/metadata fetchMetrics
v @ app-main.bundle.js?1d74d19605fd6afae319:5
app-main.bundle.js?1d74d19605fd6afae319:5 DEPRECATED: metabase/redux/metadata fetchSegments
v @ app-main.bundle.js?1d74d19605fd6afae319:5
10app-main.bundle.js?1d74d19605fd6afae319:5 Warning: can't clean query without metadata!
value @ app-main.bundle.js?1d74d19605fd6afae319:5
app-main.bundle.js?1d74d19605fd6afae319:5 Tooltip::componentDidMount: no DOM node for tooltip undefined
value @ app-main.bundle.js?1d74d19605fd6afae319:5
```

I poked around the metabase logs as well, but I don't see anything out of the ordinary there (nor would I expect to, since this seems like a front end thing)

**To Reproduce**
1. Go to Ask a question -> Custom Question
2. Choose Sample Dataset -> Orders as the data source
3. When the filter option appears, it will extend vertically beyond the top border of a small window (I'm on a 2015 15" MBP with the browser fullscreened). See screenshot below.

**Expected behavior**
I'd expect the list to either render downward and/or at least allow me to scroll to see offscreen choices.

**Screenshots**
![image](https://user-images.githubusercontent.com/4392097/62801626-2c341080-bab4-11e9-9e09-a04ab77d15af.png)

**Information about your Metabase Installation:**
- Your browser and the version: Chrome Version 75.0.3770.142 (Official Build) (64-bit)

- Your operating system: OSX 10.14.1 

- Your databases: MySQL

- Metabase version: 0.33.0-RC1, Built on 2019-07-30, Branch: release-0.33.x, Hash: 592ef43

- Metabase hosting environment: Docker

- Metabase internal database: MySQL

**Severity**
I could imagine this being a fairly substantial blocker to new people getting up to speed on notebook mode, particularly since the sample dataset field lists are much smaller than our production database's. Hopefully this is an easy fix though!

**Comment by mazameli** [comment]
Thanks Nick! This was reported in https://github.com/metabase/metabase/issues/10448 and was fixed in `release-0.33.x` by https://github.com/metabase/metabase/pull/10497

**Comment by nickhamlin** [comment]
Thanks Maz, sorry for the duplicate!
